pandas/core/series.py

```diff
@@ -2745,9 +2745,10 @@ def _take(self, indices, axis=0, convert=True, is_copy=False):
 
     def isin(self, values):
         """
-        Return a boolean :class:`~pandas.Series` showing whether each element
-        in the :class:`~pandas.Series` is exactly contained in the passed
-        sequence of ``values``.
+        Check whether ``values`` are contained in ``Series``.
+
+        Return a boolean ``Series`` showing whether each element in the
+        ``Series`` is exactly contained in the passed sequence of ``values``.
```

Comment:
consider wording as "matches an element in the passed sequence of ``values`` exactly"

Reply:
updated in c17e423